config/locales/en.yml

```diff
@@ -3938,6 +3939,7 @@ en:
     For more information on what the check provides, what data is needed to provide available updates, and how to disable this check, please visit <a href="%{more_info_url}">the configuration documentation</a>.
   text_own_membership_delete_confirmation: "You are about to remove some or all of your permissions and may no longer be able to edit this project after that.\nAre you sure you want to continue?"
   text_plugin_assets_writable: "Plugin assets directory writable"
+  text_permanent_delete_confirmation_checkbox_label: "I understand that this deletion cannot be reversed"
```

Comment:
@HDinger I think this belongs in the global I18n strings. (Not sure though if we want to reconsider in specifying a `DangerDialogComponent` default checkbox label?)

Reply:
I agree and I try to put generalizable strings into the main en.ml so they can be reused.

Reply:
Hi @myabc 

@Kharonus added the DangerDialogs to the hierarchy CF in the scope of https://github.com/opf/openproject/pull/17717. He there added the exact same strings for the button and the checkbox label. So you can rebase your PR and remove the strings here.

Reply:
@HDinger great, will do!